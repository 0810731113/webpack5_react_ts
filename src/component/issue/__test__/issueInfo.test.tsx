import React from "react";
import pretty from "pretty";
import { render, screen } from "@testing-library/react";
import { act } from "react-dom/test-utils";
import { render as renderHtml, unmountComponentAtNode } from "react-dom";

import IssueInfo from "../IssueInfo";

let container: HTMLDivElement | null;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  if (container) {
    unmountComponentAtNode(container);
    container.remove();
    container = null;
  }
});

jest.mock("react-router-dom", () => ({
  useHistory: () => ({
    push: jest.fn(),
    replace: jest.fn(),
  }),
}));

test("test issue info", () => {
  const issue = {
    id: 1725,
    title: "Fixbug/zhangm ae",
    description: undefined,
    memo: undefined,
    file:
      "https://gdc-te-public.obs.cn-north-4.myhuaweicloud.com:443/ce57677c-fffe-4843-8ca2-45cad4987c05",
    type: "团队问题",
    userId: "6742696179203941218",
    userName: "GDC21@GDCP",
    teamId: "a5d192d7-1a36-49c3-98e4-bf9835f892c3",
    teamName: "团队B",
    elementId: "2305872421149737970",
    elementName: "墙实例 [737970]",
    markCoordinate: `{"x":-3336.699993197364,"y":-1462.9634551545546,"z":1162.33513167562}`,
    issueDatasets: [
      {
        id: 1767,
        issueId: 1725,
        versionId: 38748,
        datasetId: "bf4639dd-ebc2-4718-90a6-2e2c14ab221c",
        datasetName: "ds0017777",
        teamId: "a5d192d7-1a36-49c3-98e4-bf9835f892c3",
        teamName: "团队B",
        version: 1,
        isCurrent: 1,
      },
    ],
    camera: `{"appKey":"BIM-MODEL","schemaKey":"CameraInfo","schemaVersion":1,"data":"{"cameraVersion":0,"name":"persp","fov":45,"zoom":1,"version":1,"coordinateSystem":"world","position":{"x":-12456.640180778193,"y":-7128.100346033577,"z":14663.356439086934},"target":{"x":-6060.033331145291,"y":-731.4928152042385,"z":1500.0019379108144},"up":{"x":0.5827608107570202,"y":0.5827607956395995,"z":0.5663741629976791}}"}`,
    status: "解决中",
    closeUserId: undefined,
    closeVersion: -1,
    closeUserName: undefined,
    sequenceNo: 1,
    extraKey: undefined,
    extraData: undefined,
    creationTime: "2021-02-02T08:48:40.000+0000",
    updateTime: undefined,
  };
  act(() => {
    renderHtml(
      <div>
        <IssueInfo issue={issue} />
        <IssueInfo issue={issue} isPanel />
        <IssueInfo issue={issue} hideRemark />
        <IssueInfo issue={issue} hideRemark isPanel />
      </div>,
      container,
    );
  });

  expect(pretty(container?.innerHTML)).toMatchSnapshot();
});
