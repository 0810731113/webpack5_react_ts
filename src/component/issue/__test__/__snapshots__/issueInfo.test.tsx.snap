// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test issue info 1`] = `
"<div>
  <div class=\\"ant-descriptions\\">
    <div class=\\"ant-descriptions-view\\">
      <table>
        <tbody>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题状态</span><span class=\\"ant-descriptions-item-content\\"><span class=\\"ant-tag ant-tag-red issue-status-tag\\">打开中</span> (<button type=\\"button\\" class=\\"ant-btn ant-btn-link\\"><span>查看工作流</span></button>)</span></div>
            </td>
          </tr>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题来源</span><span class=\\"ant-descriptions-item-content\\">团队问题</span></div>
            </td>
          </tr>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">工作单元</span><span class=\\"ant-descriptions-item-content\\">ds0017777 <br></span></div>
            </td>
          </tr>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">负责人</span><span class=\\"ant-descriptions-item-content\\">无</span></div>
            </td>
          </tr>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题描述</span><span class=\\"ant-descriptions-item-content\\">无</span></div>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div class=\\"ant-descriptions\\" style=\\"margin-top: 0px;\\">
    <div class=\\"ant-descriptions-view\\">
      <table>
        <tbody>
          <tr class=\\"ant-descriptions-row\\">
            <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">构件详情</span></div>
            </th>
          </tr>
          <tr class=\\"ant-descriptions-row\\">
            <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
              <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">版本</th><th class=\\"ant-table-cell\\">构件详情</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\">ds0017777</td><td class=\\"ant-table-cell\\">V1</td><td class=\\"ant-table-cell\\"><div class=\\"ant-typography\\">构件<br>墙实例 [737970]<div role=\\"button\\" tabindex=\\"0\\" class=\\"ant-typography-copy\\" aria-label=\\"Copy\\" style=\\"border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;\\"><span role=\\"img\\" aria-label=\\"copy\\" class=\\"anticon anticon-copy\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"copy\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z\\"></path></svg></span></div>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </div>
</div>
</div>
</div>
</div>
</div></span></div>
</td>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">提问场景</span></div>
  </th>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">提出时间</th><th class=\\"ant-table-cell\\">提问人</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\"><input readonly=\\"\\" style=\\"width: 60px; text-align: center;\\" type=\\"text\\" class=\\"ant-input ant-input-sm\\" value=\\"1个\\"></td><td class=\\"ant-table-cell\\">2021/02/02 16:48</td><td class=\\"ant-table-cell\\">GDC21@GDCP</td></tr></tbody></table></div></div></div></div></div></div></span></div>
  </td>
</tr>
</tbody>
</table>
</div>
</div>
<div class=\\"ant-descriptions\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题状态</span><span class=\\"ant-descriptions-item-content\\"><span class=\\"ant-tag ant-tag-red issue-status-tag\\">打开中</span> </span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题来源</span><span class=\\"ant-descriptions-item-content\\">团队问题</span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题描述</span><span class=\\"ant-descriptions-item-content\\">无</span></div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<div class=\\"ant-descriptions\\" style=\\"margin-top: 0px;\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">构件详情</span></div>
          </th>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">版本</th><th class=\\"ant-table-cell\\">构件详情</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\">ds0017777</td><td class=\\"ant-table-cell\\">V1</td><td class=\\"ant-table-cell\\"><div class=\\"ant-typography\\">构件<br>墙实例 [737970]<div role=\\"button\\" tabindex=\\"0\\" class=\\"ant-typography-copy\\" aria-label=\\"Copy\\" style=\\"border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;\\"><span role=\\"img\\" aria-label=\\"copy\\" class=\\"anticon anticon-copy\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"copy\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z\\"></path></svg></span></div>
  </div>
  </td>
  </tr>
  </tbody>
  </table>
</div>
</div>
</div>
</div>
</div>
</div></span></div>
</td>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">提问场景</span></div>
  </th>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">提出时间</th><th class=\\"ant-table-cell\\">提问人</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\"><input readonly=\\"\\" style=\\"width: 60px; text-align: center;\\" type=\\"text\\" class=\\"ant-input ant-input-sm\\" value=\\"1个\\"></td><td class=\\"ant-table-cell\\">2021/02/02 16:48</td><td class=\\"ant-table-cell\\">GDC21@GDCP</td></tr></tbody></table></div></div></div></div></div></div></span></div>
  </td>
</tr>
</tbody>
</table>
</div>
</div>
<div class=\\"ant-descriptions\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题状态</span><span class=\\"ant-descriptions-item-content\\"><span class=\\"ant-tag ant-tag-red issue-status-tag\\">打开中</span> (<button type=\\"button\\" class=\\"ant-btn ant-btn-link\\"><span>查看工作流</span></button>)</span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题来源</span><span class=\\"ant-descriptions-item-content\\">团队问题</span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">工作单元</span><span class=\\"ant-descriptions-item-content\\">ds0017777 <br></span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">负责人</span><span class=\\"ant-descriptions-item-content\\">无</span></div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<div class=\\"ant-descriptions\\" style=\\"margin-top: 0px;\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">构件详情</span></div>
          </th>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">版本</th><th class=\\"ant-table-cell\\">构件详情</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\">ds0017777</td><td class=\\"ant-table-cell\\">V1</td><td class=\\"ant-table-cell\\"><div class=\\"ant-typography\\">构件<br>墙实例 [737970]<div role=\\"button\\" tabindex=\\"0\\" class=\\"ant-typography-copy\\" aria-label=\\"Copy\\" style=\\"border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;\\"><span role=\\"img\\" aria-label=\\"copy\\" class=\\"anticon anticon-copy\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"copy\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z\\"></path></svg></span></div>
  </div>
  </td>
  </tr>
  </tbody>
  </table>
</div>
</div>
</div>
</div>
</div>
</div></span></div>
</td>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">提问场景</span></div>
  </th>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">提出时间</th><th class=\\"ant-table-cell\\">提问人</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\"><input readonly=\\"\\" style=\\"width: 60px; text-align: center;\\" type=\\"text\\" class=\\"ant-input ant-input-sm\\" value=\\"1个\\"></td><td class=\\"ant-table-cell\\">2021/02/02 16:48</td><td class=\\"ant-table-cell\\">GDC21@GDCP</td></tr></tbody></table></div></div></div></div></div></div></span></div>
  </td>
</tr>
</tbody>
</table>
</div>
</div>
<div class=\\"ant-descriptions\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题状态</span><span class=\\"ant-descriptions-item-content\\"><span class=\\"ant-tag ant-tag-red issue-status-tag\\">打开中</span> </span></div>
          </td>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">问题来源</span><span class=\\"ant-descriptions-item-content\\">团队问题</span></div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<div class=\\"ant-descriptions\\" style=\\"margin-top: 0px;\\">
  <div class=\\"ant-descriptions-view\\">
    <table>
      <tbody>
        <tr class=\\"ant-descriptions-row\\">
          <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">构件详情</span></div>
          </th>
        </tr>
        <tr class=\\"ant-descriptions-row\\">
          <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
            <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">版本</th><th class=\\"ant-table-cell\\">构件详情</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\">ds0017777</td><td class=\\"ant-table-cell\\">V1</td><td class=\\"ant-table-cell\\"><div class=\\"ant-typography\\">构件<br>墙实例 [737970]<div role=\\"button\\" tabindex=\\"0\\" class=\\"ant-typography-copy\\" aria-label=\\"Copy\\" style=\\"border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;\\"><span role=\\"img\\" aria-label=\\"copy\\" class=\\"anticon anticon-copy\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"copy\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z\\"></path></svg></span></div>
  </div>
  </td>
  </tr>
  </tbody>
  </table>
</div>
</div>
</div>
</div>
</div>
</div></span></div>
</td>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <th class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-label\\">提问场景</span></div>
  </th>
</tr>
<tr class=\\"ant-descriptions-row\\">
  <td class=\\"ant-descriptions-item\\" colspan=\\"1\\">
    <div class=\\"ant-descriptions-item-container\\"><span class=\\"ant-descriptions-item-content\\"><div class=\\"ant-table-wrapper\\" style=\\"width: 100%;\\"><div class=\\"ant-spin-nested-loading\\"><div class=\\"ant-spin-container\\"><div class=\\"ant-table\\"><div class=\\"ant-table-container\\"><div class=\\"ant-table-content\\"><table style=\\"table-layout: auto;\\"><colgroup></colgroup><thead class=\\"ant-table-thead\\"><tr><th class=\\"ant-table-cell\\">工作单元</th><th class=\\"ant-table-cell\\">提出时间</th><th class=\\"ant-table-cell\\">提问人</th></tr></thead><tbody class=\\"ant-table-tbody\\"><tr class=\\"ant-table-row ant-table-row-level-0\\"><td class=\\"ant-table-cell\\"><input readonly=\\"\\" style=\\"width: 60px; text-align: center;\\" type=\\"text\\" class=\\"ant-input ant-input-sm\\" value=\\"1个\\"></td><td class=\\"ant-table-cell\\">2021/02/02 16:48</td><td class=\\"ant-table-cell\\">GDC21@GDCP</td></tr></tbody></table></div></div></div></div></div></div></span></div>
  </td>
</tr>
</tbody>
</table>
</div>
</div>
</div>"
`;
